### PR TITLE
Add module to package

### PR DESCRIPTION
**Pull request by esparagoza-mikkakette**
Add my menu to main menu

### CODE REVIEW

File: esparagoza.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                os.system('cls')
+                print('Goals')
+                goals()
+
+                input('Press Enter to Continue')
+                print()
+            
+            case 3:
+                os.system('cls')
+                print('Comment from Alejandro:')
```
Comment (alejandro-aleckmcklaiyre) [inline esparagoza.py]: I will add my own comment here.

**Review by alejandro-aleckmcklaiyre — CHANGES_REQUESTED** [review]

File: esparagoza.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                os.system('cls')
+                print('Comments from Gomez:')
+                print()
+
+                input('Press Enter to Continue')
+                print()
+            
+            case 6:
+                os.system('cls')
+                print('Comments from Mosenos:')
```
Comment (ghost) [inline esparagoza.py]: I will add my comment here.

**Review by ghost — CHANGES_REQUESTED** [review]

File: esparagoza.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                print()
+                print("Hello, that is a nice goal. Good luck!")
+
+                input('Press Enter to Continue')
+                print()
+
+            case 4:
+                os.system('cls')
+                print('Comment from De Leon:')
+                print()
```
Comment (deleon-jasminerobelle) [inline esparagoza.py]: I will add my comment here.

**Review by deleon-jasminerobelle — CHANGES_REQUESTED** [review]

File: esparagoza.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+                input('Press Enter to Continue')
+                print()
+
+            case 5:
+                os.system('cls')
+                print('Comments from Gomez:')
+                print()
```
Comment (gomez-ashleyhermione) [inline esparagoza.py:71]: I will add my comment here

**Review by gomez-ashleyhermione — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by alejandro-aleckmcklaiyre — APPROVED** [review]

**Review by gomez-ashleyhermione — APPROVED** [review]

**Review by deleon-jasminerobelle — APPROVED** [review]

**Pull request merged**